lldb/source/API/SBFrame.cpp

```diff
@@ -1207,6 +1210,24 @@ lldb::LanguageType SBFrame::GuessLanguage() const {
   return eLanguageTypeUnknown;
 }
 
+lldb::SBStructuredData SBFrame::GetLanguageSpecificData() const {
+  auto mangled_name = GetFunction().GetMangledName();
+  if (!mangled_name)
+    return {};
+
+  auto dict_sp = std::make_shared<StructuredData::Dictionary>();
+
+  // BEGIN SWIFT
+  auto is_async =
```

Comment:
Can we detect the language of the frame, and forward this method to the corresponding language runtime, so that this code remains generic and Swift-unaware? Maybe we don't even need to figure out the language, and can just iterate over the language plugins and let each populate its own entries in the dictionary, assuming the keys are unique across languages. 

My goal is to move the abstraction in the language plugin to :

1. Avoid modifying generic code here
2. Provide the same capabilities to other languages
3. Make the generic parts of this patch upstreamable

Reply:
Implemented in b2477fd.